core_server/tasks_settings.py

```diff
@@ -24,7 +24,7 @@
         ),
     },
     "retry_failed_callbacks": {
-        "task": "analysis_module.tasks.resend_failed_callbacks",
+        "task": "core.tasks.resend_failed_callbacks",
         "schedule": crontab(
             minute="*/15",
```

Comment:
maybe pass it through env var. what do you think?

Reply:
Good point. done!